src/Microsoft.Azure.SignalR.Common/Utilities/GeneratedRestClient.cs

```diff
@@ -0,0 +1,43 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Net.Http.Headers;
+using Microsoft.Rest;
+
+namespace Microsoft.Azure.SignalR
+{
+    public partial class GeneratedRestClient
```

Comment:
a better naming, this is not Generated

Reply:
This a a partial class and one part is generated.